src/components/Switch/Switch.spec.tsx

```diff
@@ -0,0 +1,41 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { mount } from "@cypress/react";
+import { Size } from "@utilities/enum";
+import { FC, useState } from "react";
+import Switch, { SwitchProps } from "./Switch";
+import css from "./Switch.module.css";
+
+const Component: FC<SwitchProps> = ({ on = false, ...props }: SwitchProps) => {
+    const [active, setActive] = useState(on);
+
+    return <Switch {...props} on={active} onChange={() => setActive(!active)} />;
+};
+
+describe("Switch Component", () => {
+    it("should render the value correctly", () => {
+        mount(<Component name="switch-test-value" />);
+
+        cy.get("[data-test-id=Switch]").as("Switch");
+        cy.get("@Switch").invoke("attr", "name").should("eq", "switch-test-value");
+        cy.get("@Switch").should("have.value", "false");
+
+        cy.get("@Switch").click();
+        cy.get("@Switch").should("have.value", "true");
+    });
+
+    it("should do nothing if disabled", () => {
+        mount(<Component name="switch-test-disabled" disabled />);
+
+        cy.get("[data-test-id=Switch]").as("Switch");
```

Comment:
We use kebab-case for those

Reply:
https://github.com/Frontify/arcade/pull/8/commits/7366c57ecc30e14003f3251c1bad7f8a70a5e7fd I adjusted the generator so it's now kebab-case by default ;)